wiki.py

```diff
@@ -93,7 +93,7 @@ def add_project_page(
                         self._create_goal_fulfillment_text(
                             goals.keys(),
                             goal_fulfillments
-                        )
+                            )  # noqa:E123
```

Comment:
Why is this indentation off?

Reply:
Looks like I accidentally indented this. The noqa is needed even after fixing that though because the first bracket is technically on line 92 which starts one indent less.